templates/tests/server-test.yaml

```
@@ -7,6 +7,8 @@ metadata:
   annotations:
     "helm.sh/hook": test
 spec:
+  imagePullSecrets:
+    {{- toYaml .Values.global.imagePullSecrets | nindent 8 }}
```

After rebasing master:

```suggestion
  {{- include "imagePullSecrets" . | nindent 6 }}
```